src/assets/release-notes-1.15.2.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
We have to run prettier on that file to make it pass the CI. To do so, you can clone the repository and then:
```
> npm install
> npm run prettier
```
It will complain that the git tree is not clean, but when you commit the changes into the PR then everything should be fine. :)

Reply:
Ok, did that & it's passing now 
:)

Reply:
Are you able to merge @saschagrunert ?